### ISSUE TITLE
[QA] area-alt rule should have wcag412 and wcag244 tags

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2016]
The `area-alt` rule tests more success criteria than it is currently tagged for. The following tags need to be added:

- wcag412
- wcag244

**Comment by jkodu** [comment]
@jeankaplansky does this need an update to docs elsewhere?

**Comment by padmavemulapati** [comment]
Verified with the latest axe-coconut where seeing the changes for aria-alt with added tags, "wcag412
wcag244 "  - axe-coconut (v4.3.0.06000 )

![image](https://user-images.githubusercontent.com/47409579/75663584-15931800-5c97-11ea-8528-ee6cd77f2fa2.png)

**Comment by jkodu** [comment]
> Worked with DQU to update tags respectively in DQU content
